Studio-Lovelies/GG-JointJustice-Unity
# Feature/176 validate actions inside .ink files

## Summary

An attempt at #176. When an Ink file is compiled after being modified it is validated, and any errors found on actions lines are reported in the console. Also adds a menu item which allows actions in every Ink file in the project to be validated.

## Before/after screenshots and/or animated gif

<img width="679" alt="Screenshot 2022-02-14 at 09 06 29" src="https://user-images.githubusercontent.com/67964179/153833283-89a4f69b-3c36-42a6-b22a-743a34b61416.png">

## Testing instructions

Open an Ink file with errors (e.g., ActorTestInk.ink), modify it, and check that errors are reported in the Unity console when it is recompiled. Select Assets/Validate Narrative Scripts, and see that the console is filled with errors from all our broken narrative scripts.

## Additional information

- `[ ]` Changes UI
- `[x]` Introduces new feature
- `[ ]` Removes existing feature
- `[ ]` Has associated resource: 
  
closes #176
